lightgbm/src/main/scala/com/microsoft/azure/synapse/ml/lightgbm/params/LightGBMParams.scala

```diff
@@ -66,7 +66,7 @@ trait LightGBMExecutionParams extends Wrappable {
 
   val microBatchSize = new IntParam(this, "microBatchSize",
     "Specify how many elements are sent in a streaming micro-batch.")
-  setDefault(microBatchSize -> 1)
+  setDefault(microBatchSize -> 100)
```

Comment:
we should make sure that if a partition contains 70 rows or something than it doesent hang

Reply:
I know it doesn't (since there are no test datasets with an even 100 number of rows), but you are right there should be a test for it explicitly